tests/queries/0_stateless/03174_merge_join_bug.sql

```diff
@@ -0,0 +1,11 @@
+-- Tags: no-random-settings
+
+-- https://github.com/ClickHouse/ClickHouse/issues/24395
```

Comment:
Was this meant to be `https://github.com/ClickHouse/ClickHouse/issues/65611`?

Reply:
test was copied+pasted as is, let's remove it